.github/workflows/pr-packages.yaml

```
@@ -0,0 +1,21 @@
+name: Verify PR
+on:
+  pull_request:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  check-packages:
+    name: No PR preview references
+    runs-on: ${{ vars.RUNS_ON }}
```

```suggestion
    runs-on: ubuntu-latest
```
we do not run public on hosted runners and do not have var for it

---

Done